packages/library/src/providers/CFDProvider/CFDProvider.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { createContext, useCallback, useMemo, useState } from 'react';
+import { TMarketTypes, TPlatforms, THooks } from '../../types';
+
+type TCFDState = {
```

Comment:
Better to  rename it UIProvider, in the Traders hub we want to use it for demo/real, eu/non-eu , ...

Reply:
We can have another one for that IMO. CFDProvider should be scoped to its own thing. If we do this, in a few months this will be another u-store 👀

Reply:
Makes sense